### ISSUE TITLE
Small design issue in metaboxes

### ISSUE THREAD

**Issue by stevejburge** [TaxoPress/TaxoPress#1574]
Can we move this extra button and links into the main body of the metaboxes?
<img width="1370" alt="Screenshot 2023-03-20 at 11 26 57 AM" src="https://user-images.githubusercontent.com/3868028/226388516-922dfb1d-bc38-4bbd-9c35-bfd8269dc9c1.png">

**Comment by ojopaul** [comment]
@stevejburge Moving the refresh button means we're moving this dropdown as well.

So, are you giving confirmation for that?

![drop](https://user-images.githubusercontent.com/92122385/226910100-0fc0a287-4970-4eeb-8032-f28f0028e326.jpg)

**Comment by stevejburge** [comment]
@ojopaul Yes, please. Let's follow the normal WordPress standards here. It will help us avoid future issues

**Comment by stevejburge** [comment]
From @ojopaul 

> 
This should easy right? But it end up requiring more work changing the way the metabox work and action binding. For example, the button are added via javascript and when refresh is clicked, the replace whole body content. So, we’re literally reviewing and changing the process including current implementation.
